tests/robot-tests/tests/admin/bau/release_content.robot

```diff
@@ -79,35 +79,35 @@ Add Useful information related page link to release
     user clicks button    Create link
 
 Add secondary statistics
-    user waits until page does not contain element    ${SECONDARY_STATS_TABLE_TAB_SELECTOR}
+    user waits until page does not contain element    //*[@id="${SECONDARY_STATS_TABLE_TAB_SELECTOR}"]
     user clicks button    Add secondary stats
-    user waits until page contains element    css:select[name="selectedDataBlock"]    %{WAIT_MEDIUM}
-    user checks select contains x options    css:select[name="selectedDataBlock"]    5
-    user checks select contains option    css:select[name="selectedDataBlock"]    Select a data block
-    user checks select contains option    css:select[name="selectedDataBlock"]    Data Block 1
-    user checks select contains option    css:select[name="selectedDataBlock"]    Data Block 2
-    user checks select contains option    css:select[name="selectedDataBlock"]    Key Stats Data Block 1
-    user checks select contains option    css:select[name="selectedDataBlock"]    Key Stats Data Block 2
+    user waits until page contains element    //*[@name="selectedDataBlock"]    %{WAIT_MEDIUM}
+    user checks select contains x options    //*[@name="selectedDataBlock"]    5
+    user checks select contains option    //*[@name="selectedDataBlock"]    Select a data block
+    user checks select contains option    //*[@name="selectedDataBlock"]    Data Block 1
+    user checks select contains option    //*[@name="selectedDataBlock"]    Data Block 2
+    user checks select contains option    //*[@name="selectedDataBlock"]    Key Stats Data Block 1
+    user checks select contains option    //*[@name="selectedDataBlock"]    Key Stats Data Block 2
     user chooses and embeds data block    Data Block 1
```

Comment:
With all of these rows, rather than using an xpath selector you should be able to do `name:selectedDataBlock`. The same change can be made elsewhere too.